### PR TITLE
Remove some 7.7 and 7.8 transport versions

### PR DESCRIPTION
**Pull request by thecoop**

### CODE REVIEW

File: server/src/test/java/org/elasticsearch/action/OriginalIndicesTests.java [hunk 1]
```diff
@@ -43,15 +42,7 @@ public void testOriginalIndicesSerialization() throws IOException {
...
-            // indices options are not equivalent when sent to an older version and re-read due
-            // to the addition of hidden indices as expand to hidden indices is always true when
-            // read from a prior version
-            if (out.getTransportVersion().onOrAfter(TransportVersions.V_7_7_0)
-                || originalIndices.indicesOptions().expandWildcardsHidden()) {
-                assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
-            } else if (originalIndices.indicesOptions().expandWildcardsHidden()) {
-                assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
-            }
+            assertThat(originalIndices2.indicesOptions(), equalTo(originalIndices.indicesOptions()));
```
Comment (ldematte) [inline server/src/test/java/org/elasticsearch/action/OriginalIndicesTests.java:45]: Isn't this missing `if (originalIndices.indicesOptions().expandWildcardsHidden())`?

Comment (thecoop) [reply server/src/test/java/org/elasticsearch/action/OriginalIndicesTests.java:45]: The first `onOrAfter` check is always true, which means it never evaluates the second `if` clause

Comment (ldematte) [reply server/src/test/java/org/elasticsearch/action/OriginalIndicesTests.java:45]: Oh yeah, sure! Sorry about that

**Review by ldematte — APPROVED** [review]
LGTM

**Pull request merged**